packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -59,44 +47,25 @@ export const SidePanel = ({
         </Transition.Child>
 
         {/* Full-screen container to center the panel */}
-        <div
```

Comment:
Removed sliding transition for now as it has some side effects. Reverted to the old behaviour.

Reply:
worth looking into this for a new implementation of the side panel
https://ui.shadcn.com/docs/components/sheet